src/Parse/ParseObject.php

```diff
@@ -907,6 +907,8 @@ public function _encode()
         foreach ($this->estimatedData as $key => $value) {
             if (is_object($value) && $value instanceof Encodable) {
                 $out[$key] = $value->_encode();
+            } elseif (self::isAssoc($value)) {
```

Comment:
Hey sorry for the delay! I must have missed the initial opening of this PR...Anyways, to get started nice catching the issue regarding _encode! I ran your tests and can verify the observed issue.

The solution is actually much simpler than this. You won't need to perform a separate check for whether an array is associative or not. I'll explain in a few lines below.